__tests__/app.test.js

```diff
@@ -272,4 +271,32 @@ describe('POST /api/articles/:article_id/comments', () => {
             expect(body.msg).toBe("User not found")
         })
     });
+});
+
+describe('DELETE /api/comments/:comment_id', () => {
+    it('Status: 204 and no content upon succesful deletion', () => {
+        return request(app)
+        .delete('/api/comments/1')
+        .expect(204)
+        .then(({ body }) => {
```

Comment:
A `204` doesn't and shouldn't have a response body, so this test block can end on line 280. Checking status code is enough :)